app/models/school.rb

```diff
@@ -63,4 +66,8 @@ def update_searchable
         .values
         .first
   end
+
+  def name_without_punctuation
+    name.gsub(/['’.“”"]/, "").gsub(/[^0-9A-Za-z\s]/, " ")
```

Comment:
would be nice to make this method re-usable so the `SchoolSearch` can tap into it for the query

Reply:
Might be worth adding a comment for why we're doing two sorts of substitution? Examples that each are targetting?

Reply:
Added a service in https://github.com/DFE-Digital/register-trainee-teachers/pull/979/commits/7539691838cbef6431a391cda0bd61eedf636e52

Reply:
I wonder if it's named right? would `normalise` or `clean` or `sanitise` better describe what it's doing?

Reply:
Sanitise sounds good actually

Reply:
Ed you should review all of our method names